includes/Services/SiteGenService.php

```diff
@@ -890,23 +762,37 @@ public static function get_homepages() {
 	}
 
 	/**
-	 * Get the locale entered during the sitegen flow.
+	 * Get the prompt entered during the sitegen flow.
 	 *
 	 * @return string|false
 	 */
-	public static function get_locale() {
+	public static function get_prompt() {
 		$data = ReduxStateService::get( 'input' );
-		return ! empty( $data['selectedLocale'] ) ? $data['selectedLocale'] : false;
+		return ! empty( $data['prompt'] ) ? $data['prompt'] : false;
 	}
 
 	/**
-	 * Get the prompt entered during the sitegen flow.
+	 * Get the site type entered during the sitegen flow.
+	 *
+	 * @return string|false - The site type or false if not set.
+	 */
+	public static function get_site_type(): string|false {
```

Comment:
I think this might break in PHP 7.3

Reply:
Good catch! fixed here: https://github.com/newfold-labs/wp-module-onboarding-data/pull/152/commits/60c2cd47d497bb7dcae2b9d6109d0ea00dfea585